sunpy/util/sysinfo.py

```diff
@@ -81,13 +81,13 @@ def system_info():
         print(f"OS: Windows {platform.release()} {platform.version()}")
     else:
         print("Unknown OS")
-    for sys_info in ['Arch', 'SunPy']:
-        print('{} : {}'.format(sys_info, sys_prop[sys_info]))
+    for sys_info in ['Arch', 'sunpy']:
+        print(f'{sys_info}: {sys_prop[sys_info]}')
     print()
     print("Required Dependices")
```

Comment:
I also just noticed this is spelt wrong.

Reply:
Fixed